js/src/jsonrpc/interfaces/parity.js

```diff
@@ -643,6 +643,34 @@ export default {
     }
   },
 
+  removeTransaction: {
+    section: SECTION_ACCOUNTS,
+    desc: 'Cancel a pending transaction. If the transaction has already been pushed to the network, there is no guarantee that the transaction can be removed.',
+    params: [
+      {
+        type: Data,
+        desc: 'transactionId, 32-byte hex',
+        example: '0x1db2c0cf57505d0f4a3d589414f0a0025ca97421d2cd596a9486bc7e2cd2bf8b'
+      }
+    ],
+    returns: {
+      type: Object,
+      desc: 'the removed transaction if cancellation was successful, an error if not.',
```

Comment:
note: it's rather "the removed transaction if found, an error if not" (no guarantees about success state here as well)